src/StdUtils.sol

```diff
@@ -4,12 +4,12 @@ pragma solidity >=0.6.2 <0.9.0;
 import "./console2.sol";
 
 abstract contract StdUtils {
+    uint256 private constant INT256_MIN_ABS =
+        57896044618658097711785492504343953926634992332820282019728792003956564819968;
     uint256 private constant UINT256_MAX =
         115792089237316195423570985008687907853269984665640564039457584007913129639935;
 
     function _bound(uint256 x, uint256 min, uint256 max) internal pure virtual returns (uint256 result) {
-        require(min <= max, "StdUtils bound(uint256,uint256,uint256): Max is less than min.");
```

Comment:
One side effect here: by removing this `require` statement, it's no longer safe to use `_bound` as an alternative to `bound` to get the same behavior without the log noise, so this is technically a breaking change that we should avoid IMO. I think we can keep this here, and just leave the new require statement in the `boundInt` method. You get a duplicate check as a result, but that's ok

Reply:
That's a good point. I'll fix it.

Reply:
[174b694](https://github.com/foundry-rs/forge-std/pull/253/commits/174b694c8b60d2670d5e56f6d70ca554c80cc37b)